Jenkinsfile

```diff
@@ -371,7 +371,7 @@ core_logic: {
         ws('workspace/build-cmake-gpu') {
           timeout(time: max_time, unit: 'MINUTES') {
             utils.init_git()
-            utils.docker_run('ubuntu_gpu', 'build_ubuntu_gpu_cmake', false)
+            utils.docker_run('ubuntu_gpu_cu91', 'build_ubuntu_gpu_cmake', false)
```

Comment:
Why do CU91 here? Scala do CU92 as a default

Reply:
Unfortunately, until very recently (#14513), CU91 was the version CI used (with the exception of the Scala stuff you guys added). I'm sticking with CU91 because that's what `ubuntu_gpu` is in this branch [i.e.](https://github.com/apache/incubator-mxnet/blob/v1.4.x/ci/docker/Dockerfile.build.ubuntu_gpu#L21).

CI in this branch definitely deserves an update. It should be brought to CUDA 10 (and once stu1130 is finished getting cudnn 7.5. working, CUDA 10.1). But I think this requires more time than I have atm.